docs/user_workloads/instancetypes.md

```diff
@@ -503,6 +503,22 @@ null
 null
 ```
 
+## Hotplug
+
+Support for instance type based vCPU and memory hotplug was introduced in KubeVirt 1.3 and is built on existing [vCPU](../compute/cpu_hotplug.md) hotplug, [memory](../compute/memory_hotplug.md) hotplug and [LiveUpdate](./vm_rollout_strategies.md) support.
```

Comment:
```suggestion
Support for instance-type-based vCPU and memory hotplug was introduced in KubeVirt 1.3 and is built on existing [vCPU](../compute/cpu_hotplug.md) hotplug, [memory](../compute/memory_hotplug.md) hotplug and [LiveUpdate](../user-workloads/vm_rollout_strategies.md) support.
```
Why? Adding hyphens for the compound adjective helps the reader understand that these are all connected as a large adjective and none of them are a noun. It's a big nit, since they're reading all about instance types so it should be obvious but hyphens never hurt. 
Adding relativity to the link just because it helps if we ever need to move content between sections.